controller/rentBook.ctrl.js

```diff
@@ -0,0 +1,29 @@
+app.controller('rentBook', ['$scope', 'rentBookService', 'dataService', 'localStorageService', function ($scope, rentBookService, dataService, localStorageService) {
+    $scope.arr = [];
+    const a = 3;
+    $scope.currentRentBookData = rentBookService.getCurrentBook();
```

Comment:
Use currentBookSvc which will contain the currBook et from bookCtrl via currBook obj as resolve